src/implementations/twig/components/spinner/spinner.html.twig

```diff
@@ -0,0 +1,82 @@
+{% spaceless %}
+{#
+  Parameters:
+  - "variant" (string) options:
+    [ primary, negative ] (default: primary)
+  - "size" (string) options:
+    [ small, medium, large ]
+  - "text" (string) (default: '')
+  - "visible" (boolean) (default false)
+  - "extra_classes" (string) (default: '')
+  - "extra_attributes" (array) (default: []): format: [
+      {
+        "name" (string) (default: ''),
+        "value" (optional) (string)
+      },
+      ...
+    ],
+#}
+
+{# Internal properties #}
+
+{% set _css_classes = 'ecl-spinner' %}
+{% set _variant = variant|default('primary') %}
+{% set _size = size|default('medium') %}
+{% set _centered = centered|default(false) %}
+{% set _text = text|default('') %}
+{% set _overlay = overlay|default(false) %}
+{% set _overlay_classes = 'ecl-spinner__overlay' %}
+{% set _visible = visible|default(false) %}
+{% set _extra_attributes = '' %}
+
+{% set _css_classes = _css_classes ~ ' ' ~ 'ecl-spinner--' ~ _variant  ~ ' ecl-spinner--' ~ _size %}
+
+{% if _centered %}
+  {% set _css_classes = _css_classes ~ ' ' ~ 'ecl-spinner--centered' %}
+{% endif %}
+
+{% if _visible %}
+ {% set _css_classes = _css_classes ~ ' ecl-spinner--active' %}
```

Comment:
as the parameter is called "visible", maybe we could use the same name for the class (".ecl-spinner--visible"). Same for the overlay